### PR TITLE
Hound fixes

### PR DESCRIPTION
**Pull request by SoftwareDev**
Addressing Hound issues from https://github.com/MarkUsProject/Markus/pull/1954

### CODE REVIEW

**Comment by SoftwareDev** [comment]
Did some general quoting cleanup while I was at it.

File: lib/repo/git_repository.rb [hunk 1]
```diff
@@ -74,8 +78,8 @@ def self.create(connect_string)
       index = repo.index
       index.add(path: 'README.md', oid: oid, mode: 0100644)
       index.write
-      Rugged::Commit.create(repo,
-                            commit_options(repo, 'Markus',
+      Rugged::Commit.create(
+          repo, commit_options(repo, 'Markus',
                                            'Initial commit and add readme.'))
```
Comment (david-yz-liu) [inline lib/repo/git_repository.rb]: @SoftwareDev The indentation of this line is now off.

Comment (SoftwareDev) [inline lib/repo/git_repository.rb]: Ah right.

**Comment by SoftwareDev** [comment]
@david-yz-liu I've tamed the hound.

**Comment by david-yz-liu** [comment]
Watching that was really funny.

**Pull request merged**